pkg/deployments/deployments_test.go

```diff
@@ -49,6 +51,7 @@ func TestCreateDeployments(t *testing.T) {
 			tempDirPath:  "charts/templates",
 			tempFileName: "charts/templates/deployment.yaml",
 			tempPath:     "../../test/templates/helm/charts/templates/deployment.yaml",
+			fixturePath:  "../fixtures/deployments/charts/templates/deployment.yaml",
```

Comment:
why are we testing the generated deployment file against a template? shouldn't it be tested against a deployment fixture?